lib/streamlit/elements/alert.py

```diff
@@ -13,14 +13,17 @@
 # limitations under the License.
 
 from typing import cast
+from typing import TYPE_CHECKING
```

Comment:
nit: This isn't done totally consistently within the codebase (I do think it's the more common import style, though), but I'd have a slight preference for combining all of the `typing` imports into one line.

Same comment through out this PR

Reply:
I don't have strong feelings on this. Updated the PR to align with the style you suggested.